libbech32/bech32.cpp

```diff
@@ -7,15 +7,15 @@ namespace {
     using namespace bech32::limits;
 
     // exponent used in checksum generation, taken from recommendation
-    // in: https://gist.github.com/sipa/a9845b37c1b298a7301c33a04090b2eb
-    const unsigned int M = 0x3FFFFFFF;
+    // in: https://github.com/sipa/bips/blob/bip-bech32m/bip-bech32m.mediawiki
```

Comment:
link broken.  ready to point to bip350.
  https://github.com/bitcoin/bips/blob/master/bip-0350.mediawiki

sipa's working blob has moved here (but may be already abandoned):
  https://github.com/sipa/bips/blob/6446f2af0a75e513a0c4bad0ddaad8d798bd5e2d/bip-0350.mediawiki